Mathlib/Data/Real/EReal.lean

```diff
@@ -445,6 +445,144 @@ theorem eq_bot_iff_forall_lt (x : EReal) : x = ⊥ ↔ ∀ y : ℝ, x < (y : ERe
     exact ⟨x.toReal, coe_toReal_le h⟩
 #align ereal.eq_bot_iff_forall_lt EReal.eq_bot_iff_forall_lt
 
+/-! ### Intervals and coercion from reals -/
+
+lemma exists_between_ofReal {x z : EReal} (h : x < z) : ∃ y : ℝ, x < y ∧ y < z := by
```

Comment:
```suggestion
lemma exists_between_coe {x z : EReal} (h : x < z) : ∃ y : ℝ, x < y ∧ y < z := by
```
or
```suggestion
lemma exists_between_coe_real {x z : EReal} (h : x < z) : ∃ y : ℝ, x < y ∧ y < z := by
```
since the function is called `Real.toEReal`, not `EReal.ofReal`.